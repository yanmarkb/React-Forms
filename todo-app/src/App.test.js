import React from "react";
import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders task label", () => {
	render(<App />);
	const labelElement = screen.getByText(/Task:/i);
	expect(labelElement).toBeInTheDocument();
});

test("App component renders without crashing", () => {
	render(<App />);
});

test("App component matches snapshot", () => {
	const { asFragment } = render(<App />);
	expect(asFragment()).toMatchSnapshot();
});
