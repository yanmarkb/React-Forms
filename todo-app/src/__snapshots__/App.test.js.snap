// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          for="task"
        >
          Task:
        </label>
        <input
          id="task"
          name="task"
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Add Todo
        </button>
      </form>
      <div />
    </div>
  </div>
</DocumentFragment>
`;
